src/ex1/script.js

```diff
@@ -68,16 +83,17 @@ function deleteOrCompleteTask(event) {
 // filtering completed task or uncompleted task
 function filterTodo(event) {
     todos.forEach(function (todo) {
+        const hasTaskCompleted = todo.classList.contains("completed")
```

Comment:
good reuse. what I meant in the original comment is that you can take the `"completed"` value to a constant and reuse it in other places in the code

```suggestion
        const hasTaskCompleted = todo.classList.contains(TASK_COMPLETED_CLASS_NAME)
```